### PR TITLE
Manifest: Add Xiaomi Redmi Note 5 Pro/AI [ Whyred ]

### PR DESCRIPTION
**Pull request by dopaemon**
CPU : Snapdragon 636
GPU : Adreno 512
Linux : 4.4.y
Android : 7.1 / 8.1 / 9.0 / 10.0 / 11

### CODE REVIEW

**Comment by Vince1171** [comment]
I suggest to regroup all commits in one named `manifests: add Xiaomi Redmi Note 5 Pro/AI (Whyred)`

**Comment by dopaemon** [comment]
okay i will make now

Vào 18:12, Th 4, 10 thg 2, 2021 Vince1171 <notifications@github.com> đã
viết:

**Comment by dopaemon** [comment]
> I suggest to regroup all commits in one named `manifests: add Xiaomi Redmi Note 5 Pro/AI (Whyred)`

Okay i Will make now

File: manifests/xiaomi_whyred.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+          
+  <remote name="halium-9.0"
+          fetch="https://github.com/Linux-On-Sdm6Series"
+          revision="halium-9.0" />
+ 
+  <!-- Halium device For Whyred -->
+  <project path="device/halium/whyred" name="device_halium_whyred" remote="halium-9.0" />
```
Comment (Vince1171) [inline manifests/xiaomi_whyred.xml]: oh I just noticed
do not put `halium` in the `path`
put the vendor name (wiaomi here)
so `path="device/xiaomi/whyred"`
same for vendor and kernel
(put that again, in one commit plz :) )

**Review by Vince1171 — CHANGES_REQUESTED** [review]

**Pull request merged**